src/app/tests/TestCommandInteraction.cpp

```diff
@@ -87,6 +87,19 @@ constexpr CommandId kTestCommandIdFillResponseMessage     = 7;
 constexpr CommandId kTestNonExistCommandId                = 0;
 
 const app::CommandSender::TestOnlyMarker kCommandSenderTestOnlyMarker;
+
+class SimpleTLVPayload : public chip::app::DataModel::EncodableToTLV
```

Comment:
```suggestion
class SimpleTLVPayload : public app::DataModel::EncodableToTLV
```
This is inside namespace chip.